portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Bears Battlestar Gallactica', 'Identity theft is not a joke Jim!!', 'At least Im not a book face', 'Pam Pam Pam PAm'];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
```

Comment:
Given that you've removed the reference to this file in index.html, probably should just remove this file entirely.